src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import Navigation from "../../components/navigation/Navigation";
+
+function Subreddit() {
+    const [post, setPost] = useState([]);
```

Comment:
`post` zal uiteindelijk een object worden. Ik zou daarom kiezen om `undefined` als default waarde te pakken, dan heb je een duidelijker onderscheid of het al wél of niet is ingeladen
```suggestion
    const [post, setPost] = useState();
```